build/run-e2e-suite.sh

```diff
@@ -88,3 +88,10 @@ kubectl run --rm \
   --overrides='{ "apiVersion": "v1", "spec":{"serviceAccountName": "ingress-nginx-e2e"}}' \
   e2e --image=nginx-ingress-controller:e2e
 
+pwd
+hostname
+echo "extracting reports ..."
```

Comment:
We can get by labels here, interact over data values map and use the key name as the file name. 

Eg:
* k get cm -l content=report
* Use jq to iteract over the list
* iteract over data of each cm
* Key = filename, value = data